endpoint/endpoint.go

```diff
@@ -58,5 +74,5 @@ func (e *Endpoint) MergeLabels(labels map[string]string) {
 }
 
 func (e *Endpoint) String() string {
-	return fmt.Sprintf(`%s -> %s (type "%s")`, e.DNSName, e.Target, e.RecordType)
+	return fmt.Sprintf("%s %d IN %s %s", e.DNSName, e.RecordTTL, e.RecordType, e.Target)
```

Comment:
@ideahitme @linki I recommended this change to have nicer string representation, that matches the standard bind zone format, which is known to most of DNS admins.

Reply:
@linki is this a blocker or not?

Reply:
we discussed that this is ok